marker_size: "0.04"
robot: "baxter"
baxter_arm: "left"
camera: "kinect_freenect"
number_of_points: "20"
